LCM/scripts/RemoveModule.py

```diff
@@ -50,6 +50,37 @@ def RemoveMirroredDirectory(moduleDir, omiDir):
 
 print("Removing module " + moduleName)
 
+# special section for nxOMSAutomationWorker module
+# the Linux Hybrid worker needs to be killed before the module is removed
+# also a good idea to remove the state.conf file
+# we preserve the worker.conf file in case the user wants to preserve his settings
```

Comment:
Does the OMSAgent/DSC create the worker.conf file? Because if so, we should probably delete it.